tests/features/keep_using_form.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Finish by hand
```

Comment:
The description of this tests sounds like it's for finishing the form with the tool, not finishing by hand. If I've understood correctly, this might need a different name. Also, does the `Feature` value need to be unique, like an ID, or does it not matter?